webapp/templates/jury/partials/external_contest_source_info.html.twig

```diff
@@ -34,7 +34,14 @@
             {% if externalContestSource.password %}
                 <tr>
                     <th>Password</th>
-                    <td><code>{{ externalContestSource.password }}</code></td>
+                    <td>
+                        <code data-password-hidden>
+                            {% for i in 0..(externalContestSource.password|length) %}•{% endfor %}
```

Comment:
Why give away the password length? Setting just 8 symbols is good enough to indicate that its set.